frontend/src/components/MapPage/MapPage.js

```diff
@@ -38,7 +38,7 @@ export const MapPage = ({ selectLocationAndDate }) => {
     return Math.abs(
       moment(fromDate, "YYYY-MM-DD")
         .startOf("day")
-        .diff(moment(toDate, "YYYY-MM-DD").startOf("day"), "days")
+        .diff(moment(toDate, "YYYY-MM-DD").add('days', 1).startOf("day"), "days")
```

Comment:
Before: If you select range of March 27th -> March 28th this would return just March 27th
After: returns March 27th and 28th

Before: If you select March 27th -> March 27th this would error
After: returns and predicts on 27th